### ISSUE TITLE
node_modules/agm-overlays: Appears to be a git repo or submodule

### ISSUE THREAD

**Issue by mruknowme** [AckerApple/agm-overlays#31]
Getting this error after installing new agm-overlays, reverting doesn't help. Now any npm command is not executable successfully.

```
node_modules/agm-overlays: Appears to be a git repo or submodule

npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

node 12.2.0
npm 6.9.0

**Comment by havo07** [comment]
It is a topic with your team, I update without problem, try to remove the module and add it again

**Comment by AckerApple** [comment]
@havo07 is most like right. That errors comes from making manual changes to a folder. Please delete agm-overlays in node_modules and reinstall

@havo07 your help is much appreciated and I will make you a contributor os this package real soon (off to take kid to school)

**Comment by naveenkumarsangi** [comment]
I'm facing the same error on fresh install of `agm-overlays` in travis CI

**Comment by AckerApple** [comment]
@nkprince007, can you indicate if you've used this library in the past?

Really thinking updates to Angular and/or TypeScript are a part of this

**Comment by naveenkumarsangi** [comment]
No, I'm a fresh user to this package, but I've heard from my team that they've been using it for over a year.

**Comment by AckerApple** [comment]
I found it when it happened to me.

Somehow the .`git` folder got published to npm. I had no `.npmignore` file.

This is fixed in v1.4.1

If you are getting this issue from an older version, remove the folder `node_module/agm-overlays` and reinstall.
